packages/medusa/src/api/routes/admin/order-edits/add-line-item.ts

```diff
@@ -0,0 +1,102 @@
+import { Request, Response } from "express"
+import { IsInt, IsOptional, IsString } from "class-validator"
+import { EntityManager } from "typeorm"
+
+import { OrderEditService } from "../../../../services"
+import {
+  defaultOrderEditFields,
+  defaultOrderEditRelations,
+} from "../../../../types/order-edit"
+
+/**
+ * @oas [post] /order-edits/:id/items
```

Comment:
```suggestion
 * @oas [post] /order-edits/{id}/items
```

and param should be documented